### PR TITLE
WIP: addition of trino airflow scheduler

### PR DESCRIPTION
**Pull request by SiddiqueAhmad**

### CODE REVIEW

**Comment by SiddiqueAhmad** [comment]
> Thanks for contributing this @SiddiqueAhmad! Were you able to test this and verify it works?

Most welcome, and yes it is working and running in my local setup.

File: docs/integrations/engines/trino.md [hunk 1]
```diff
@@ -65,6 +65,29 @@ The `jdbc`, `rest` and `nessie` catalogs do not support views and are thus incom
...
+
+The operator requires an [Airflow connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html) to determine the target Trino account. Refer to [Trino connection](https://airflow.apache.org/docs/apache-airflow-providers-postgres/stable/connections/postgres.html) for more details.
+
+By default, the connection ID is set to `trino_default`, but can be overridden using the `engine_operator_args` parameter to the `SQLMeshAirflow` instance as in the example below:
+```python linenums="1"
+sqlmesh_airflow = SQLMeshAirflow(
+    "trino",
+    default_catalog="<database name>",
+    engine_operator_args={
+        "tromp_conn_id": "<Connection ID>"
```
[the remark this answers is not in the document]
Comment (SiddiqueAhmad) [reply docs/integrations/engines/trino.md:87]: Sending fix

File: docs/integrations/engines/trino.md [hunk 2]
```diff
@@ -65,6 +65,29 @@ The `jdbc`, `rest` and `nessie` catalogs do not support views and are thus incom
 | `retries`            | Number of retries to attempt when a request fails. Default: `3`                                                                                                           |  int   |    N     |
 | `timezone`           | Timezone to use for the connection. Default: client-side local timezone                                                                                                   | string |    N     |
 
+## Airflow Scheduler
+**Engine Name:** `trino`
+
+The SQLMesh Trino Operator is similar to the [TrinoOperator](https://airflow.apache.org/docs/apache-airflow-providers-postgres/stable/_api/airflow/providers/postgres/operators/postgres/index.html), and relies on the same [TrinoHook](https://airflow.apache.org/docs/apache-airflow-providers-postgres/stable/_api/airflow/providers/postgres/hooks/postgres/index.html) implementation.
```
[the remark this answers is not in the document]
Comment (SiddiqueAhmad) [reply docs/integrations/engines/trino.md:71]: Oh you got the source too :), yes copied let me correct it

File: docs/integrations/engines/trino.md [hunk 3]
```diff
@@ -65,6 +65,29 @@ The `jdbc`, `rest` and `nessie` catalogs do not support views and are thus incom
...
+
+To enable support for this operator, the Airflow Trino provider package should be installed on the target Airflow cluster along with SQLMesh with the Trino extra:
+```
+pip install "apache-airflow-providers-trino"
+pip install "sqlmesh[trino]"
+```
+
+The operator requires an [Airflow connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html) to determine the target Trino account. Refer to [Trino connection](https://airflow.apache.org/docs/apache-airflow-providers-trino/stable/connections.html) for more details.
```
Comment (eakmanrq) [inline docs/integrations/engines/trino.md:79]: I see you fixed this link but you still have the links above this that are linking to Postgres.

**Review by eakmanrq — APPROVED** [review]
Thanks @SiddiqueAhmad! Will fix minor issues with links in docs in follow up PR.

**Comment by SiddiqueAhmad** [comment]
I updated docs anyway thanks for merging and fixing too

**Pull request merged**